docs/install/install-client.rst

```diff
@@ -0,0 +1,134 @@
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 2
+
+.. _installing-flocker-cli-ubuntu-15.04:
+
+Ubuntu 15.04
+============
+
+On Ubuntu 15.04, the Flocker CLI can be installed from the ClusterHQ repository:
+
+.. task:: install_cli ubuntu-15.04
+   :prompt: alice@mercury:~$
+
+.. _installing-flocker-cli-ubuntu-14.04:
+
+Ubuntu 14.04
+============
+
+On Ubuntu 14.04, the Flocker CLI can be installed from the ClusterHQ repository:
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: alice@mercury:~$
+
+Other Linux Distributions
+=========================
+
+.. warning::
+
+   These are guidelines for installing Flocker on a Linux distribution which we do not provide native packages for.
+   These guidelines may require some tweaks, depending on the details of the Linux distribution in use.
+
+Before you install ``flocker-cli`` you will need a compiler, Python 2.7, and the ``virtualenv`` Python utility installed.
+
+To install these with the ``yum`` package manager, run:
+
+.. prompt:: bash alice@mercury:~$
+
+   sudo yum install gcc python python-devel python-virtualenv libffi-devel openssl-devel
+
+To install these with ``apt``, run:
+
+.. prompt:: bash alice@mercury:~$
+
+   sudo apt-get update
+   sudo apt-get install gcc libssl-dev libffi-dev python2.7 python-virtualenv python2.7-dev
+
+Then run the following script to install ``flocker-cli``:
+
+:version-download:`linux-install.sh.template`
+
+.. version-literalinclude:: linux-install.sh.template
+   :language: sh
+
+Save the script to a file and then run it:
+
+.. prompt:: bash alice@mercury:~$
+
+   sh linux-install.sh
+
+The ``flocker-deploy`` command line program will now be available in :file:`flocker-tutorial/bin/`:
+
+.. version-code-block:: console
+
+   alice@mercury:~$ cd flocker-tutorial
+   alice@mercury:~/flocker-tutorial$ bin/flocker-deploy --version
+   |latest-installable|
+   alice@mercury:~/flocker-tutorial$
+
+If you want to omit the prefix path you can add the appropriate directory to your ``$PATH``.
+You'll need to do this every time you start a new shell.
+
+.. version-code-block:: console
+
+   alice@mercury:~/flocker-tutorial$ export PATH="${PATH:+${PATH}:}${PWD}/bin"
+   alice@mercury:~/flocker-tutorial$ flocker-deploy --version
+   |latest-installable|
+   alice@mercury:~/flocker-tutorial$
+
+OS X
+====
+
+Install the `Homebrew`_ package manager.
+
+Make sure Homebrew has no issues:
+
+.. prompt:: bash alice@mercury:~$
+
+   brew doctor
+
+Fix anything which ``brew doctor`` recommends that you fix by following the instructions it outputs.
+
+Add the ``ClusterHQ/tap`` tap to Homebrew and install ``flocker``:
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: alice@mercury:~$
+
+You can see the Homebrew recipe in the `homebrew-tap`_ repository.
+
+The ``flocker-deploy`` command line program will now be available:
+
+.. version-code-block:: console
+
+   alice@mercury:~$ flocker-deploy --version
+   |latest-installable|
+   alice@mercury:~$
+
```

Comment:
Add a Next Step section, to guide the user to continue.  Note, the order will be:
```
Flocker CLI installation -> Flocker Node installation -> Client authentication -> MongoDB tutorial
```
It might make more sense to have:
```
Flocker CLI installation -> Client authentication -> Flocker Node installation  -> MongoDB tutorial
```
or perhaps add Client authentication to the reasonably short CLI installation page.